docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,151 @@
+* **Register and enroll node identities**. Just as org admin identity is registered and enrolled, the identity of a node must be registered and enrolled by the TLS CA. In this case, however, instead of giving an identity a role of `admin` or `user`, it will be given a role of `peer` or `orderer`. As with the admin, attributes and affiliations for this identity can also be assigned. The MSP structure for a node is known as a "local MSP", since the permissions assigned to the identities are only relevant at the local (node) level. It also contains a slightly different structure, since in addition to the usual root CA certificates and user identities, it also contains TLS certificates.
+
+For more conceptual information about identities and permissions in a Fabric-based blockchain network, see [Identity](identity/identity.html) and [Membership](membership/membership.html).
+
+For a look at how to use a CA to generate an admin identity and MSP, check out [Enroll Org1's Admin](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-org1-s-admin).
+
+To see how to use the TLS CA to generate the certificates and local MSP of a node, check out [Setup Org1's Peers](https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#setup-org1-s-peers).
+
+## Step five: Deploy nodes
+
+Once you have gathered all of the certificates and MSPs you need, you're almost ready to create a node. As discussed above, there are a number of valid ways to deploy nodes. In the Hyperledger Fabric documentation, we presume a Docker deployment and will, where appropriate, give sample Docker commands.
+
+### Create a peer
+
+Before you can create a peer, you will need to customize the configuration file for the peer. In Fabric, this file is known as `core.yaml`. You can find a sample `core.yaml` configuration file [in the sampleconfig directory of Hyperledger Fabric](https://github.com/hyperledger/fabric/blob/master/sampleconfig/core.yaml).
+
+As you can see in the file, there are quite a number of parameters you either have the option to set or will need to set for your node to work properly. In general, if you do not have the need to change a tuning value, leave it alone. You will, however, likely need to adjust the various addresses, specify the database type you want to use, as well as to specify where the MSP for the node is located.
+
+You have two main options for tuning your configuration.
+
+1. Edit the YAML file bundled with the binaries.
+2. Use environment variable overrides when deploying (if using Docker).
```

Comment:
This is a nice write up https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/users-guide.html#configuration-settings that we might be able to re-purpose here. What do you think?